app/controllers/tasks_controller.rb

```diff
@@ -19,4 +25,24 @@ def stop
   def complete
     render json: {}, status: :ok
   end
+
+  private
+
+  def render_tasks(tasks)
+    render json: {
+      tasks: tasks.map { |task| build_task_json task }
+    }, status: :ok
+  end
+
+  def build_task_json(task)
+    {
+      id: task.id,
+      status: 'recording',
+      startAt: task.start_at.rfc3339,
+      endAt: task.end_at&.rfc3339 || '0000-00-00T00:00:00Z',
+      duration: task.duration,
+      taskGroupId: task.task_category.id,
+      taskCategoryId: task.task_category.task_group.id
```

Comment:
```suggestion
      taskGroupId: task.task_category.task_group.id,
      taskCategoryId: task.task_category.id
```
キーと値が一致していないような気がしました。
正しそうか確認をお願いできればと思います。

Reply:
Oh... なんでこんなことに。
ありがとうございます。ほかにもあるかもなので一通り見てみます！